serviceerror/convert.go

```diff
@@ -94,7 +94,7 @@ func FromStatus(st *status.Status) error {
 		case *failure.MultiOperationExecutionAborted:
 			return newMultiOperationAborted(st)
 		default:
-			// fall through to st.Err()
+			return newAborted(st)
```

Comment:
What about out of range and unauthenticated? Should we do those now too or wait?